src/main/java/org/jasig/portlet/newsreader/adapter/RomeAdapter.java

```diff
@@ -31,8 +31,8 @@
 import net.sf.ehcache.Element;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.StringUtils;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
Yes, there is a tool that does the import replacement. Unfortunately, it does not handle {}, but since the tool does it in a second it seemed worth the effort.